modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/BasicInfoContainer.tsx

```diff
@@ -3,7 +3,9 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+
```

Comment:
@larissacribeiro Can you please squash this commit into the previous one? Also, I don't think this extra blank line here is necessary. Thanks!

Reply:
Yes, @naraandrade, just did it! Can you take another look? Cause for some reason the objectDefinition was causing an `duplicate identifier 'objectDefinition' error`;